### PR TITLE
bug #95506 - Bulk Upload of Users - MDO - Wrong error message is disp…

### PR DESCRIPTION
**Pull request by Sahil-tarento**
…laying the uploaded excel when downloaded for invalid full name input

### CODE REVIEW

File: src/main/java/org/sunbird/common/util/Constants.java [hunk 1]
```diff
@@ -693,6 +693,14 @@ public class Constants {
 	public static final String PROFILE_APPROVAL_FIELDS_KEY = "profileApprovalFields";
 	public static final String BULK_UPLOAD_VERIFICATION_REGEX = "bulk.upload.tag.verification.regex";
 
+	public static final String USER_BULK_UPLOAD_FULLNAME_ERROR_MSG = "Invalid value for Full Name. Expecting only alphabets.";
+	public static final String USER_BULK_UPLOAD_EMAIL_ERROR_MSG = "Invalid value for Email. Expecting only alphanumeric.";
+	public static final String USER_BULK_UPLOAD_PHONE_ERROR_MSG = "Invalid value for Phone. Expecting only numeric.";
+	public static final String USER_BULK_UPLOAD_GROUP_ERROR_MSG = "Invalid value for Group. Expecting only alphabets.";
+	public static final String USER_BULK_UPLOAD_TAGS_ERROR_MSG = "Invalid value for Tags. Expecting only alphabets.";
```
Comment (karthik-tarento) [inline src/main/java/org/sunbird/common/util/Constants.java]: Update the error and use the this value --> Invalid value for Tag. Tags are comma separated string values. A Tag can contain only alphabets with spaces. eg: Bihar Circle, Patna Division

File: src/main/java/org/sunbird/profile/service/UserBulkUploadService.java [hunk 2]
```diff
@@ -198,7 +198,7 @@ private void processBulkUpload(HashMap<String, String> inputDataMap) throws IOEx
                                 invalidErrList.add("Invalid Tag : Tags are comma seperated string values. A Tag can contain only alphabets with spaces. eg: Bihar Circle, Patna Division");
```
Comment (karthik-tarento) [inline src/main/java/org/sunbird/profile/service/UserBulkUploadService.java]: Use the constant here too...

File: src/main/java/org/sunbird/common/util/Constants.java [hunk 3]
```diff
@@ -693,6 +693,14 @@ public class Constants {
...
+	public static final String USER_BULK_UPLOAD_EXTERNAL_SYSTEM_ID_ERROR_MSG = "Invalid value for External System ID. Expecting only alphanumeric.";
```
Comment (karthik-tarento) [inline src/main/java/org/sunbird/common/util/Constants.java]: Update the error and use this value -> Invalid value. External System Id can contain only alphanumeric characters and have a max length of 30.

File: src/main/java/org/sunbird/profile/service/UserBulkUploadService.java [hunk 4]
```diff
@@ -213,7 +213,7 @@ private void processBulkUpload(HashMap<String, String> inputDataMap) throws IOEx
                                 invalidErrList.add("Invalid External System ID : External System Id can contain alphanumeric characters and have a max length of 30");
```
Comment (karthik-tarento) [inline src/main/java/org/sunbird/profile/service/UserBulkUploadService.java]: Use the constant here also.

File: src/main/java/org/sunbird/common/util/Constants.java [hunk 5]
```diff
@@ -693,6 +693,14 @@ public class Constants {
...
+	public static final String USER_BULK_UPLOAD_EXTERNAL_SYSTEM_ERROR_MSG = "Invalid value for External System. Expecting only alphabets.";
```
Comment (karthik-tarento) [inline src/main/java/org/sunbird/common/util/Constants.java]: Update the error and use this value -> Invalid value. External System can contain only alphabets and can have a max length of 255.

File: src/main/java/org/sunbird/profile/service/UserBulkUploadService.java [hunk 6]
```diff
@@ -223,7 +223,7 @@ private void processBulkUpload(HashMap<String, String> inputDataMap) throws IOEx
                                 invalidErrList.add("Invalid External System : External System can contain only alphabets and can have a max length of 255");
```
Comment (karthik-tarento) [inline src/main/java/org/sunbird/profile/service/UserBulkUploadService.java]: Use the constant here also

**Review by karthik-tarento — CHANGES_REQUESTED** [review]

File: src/main/java/org/sunbird/common/util/Constants.java (hunk 1 above)
Comment (Sahil-tarento) [reply src/main/java/org/sunbird/common/util/Constants.java]: I have made the changes

File: src/main/java/org/sunbird/profile/service/UserBulkUploadService.java (hunk 2 above)
Comment (Sahil-tarento) [reply src/main/java/org/sunbird/profile/service/UserBulkUploadService.java]: I have made the changes

File: src/main/java/org/sunbird/common/util/Constants.java (hunk 3 above)
Comment (Sahil-tarento) [reply src/main/java/org/sunbird/common/util/Constants.java]: I have made the changes

File: src/main/java/org/sunbird/profile/service/UserBulkUploadService.java (hunk 4 above)
Comment (Sahil-tarento) [reply src/main/java/org/sunbird/profile/service/UserBulkUploadService.java]: I have made the changes

File: src/main/java/org/sunbird/common/util/Constants.java (hunk 5 above)
Comment (Sahil-tarento) [reply src/main/java/org/sunbird/common/util/Constants.java]: I have made the changes

File: src/main/java/org/sunbird/profile/service/UserBulkUploadService.java (hunk 6 above)
Comment (Sahil-tarento) [reply src/main/java/org/sunbird/profile/service/UserBulkUploadService.java]: I have made the changes

**Pull request closed without merging**